pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -229,6 +229,10 @@ def on_run_end(self) -> List[List[STEP_OUTPUT]]:
 
         self.update_lr_schedulers("epoch", update_plateau_schedulers=True)
 
+        self.batch_progress.current.reset()
+        self.scheduler_progress.current.reset()
+        self.batch_loop.optim_progress.reset_on_epoch()
```

Comment:
same as in reset() above line 90-93

Reply:
Probably worth extracting into a protected method